docs/getting-started/mainnet.md

```diff
@@ -3,18 +3,28 @@
 
 This is a detailed documentation for setting up a **Full Node** on Crypto.org mainnet. Note that while anyone can set up a full node, only the top 100 validators are considered "active" and eligible to receive rewards. See [FAQs](https://github.com/crypto-org-chain/chain-main/discussions/442) for more info.
 
-## Step 0 : Notes on  "Canis Major" and "DRACO II" Network upgrades
+## Step 0 : Notes on network upgrades
 
-Before we start, please note that there were "*Canis Major*" and "*DRACO II*" network upgrade at the block height `922,363` and `3,526,800`, which requires the node operator to update their Crypto.org Chain Mainnet binary `chain-maind` from `v1.*.*` to `v2.0.1` then `v3.3.2`. 
+For the host who would like to build a **Run a Full Node with complete blockchain data** from scratch,
+note that there were serveral breaking network upgrades, requiring upgrading at designated block heights below:
 
-For the host who would like to build a **Full Node** with complete blockchain data from scratch, one would need to:
-1. Start the node with the older binary version `v1.2.1`; 
-1. Sync-up with the blockchain until it reaches the target upgrade block height `922,363`;
-1. Update the binary to `v2.0.1` and start the node;
-1. Sync-up with the blockchain again until it reaches the target upgrade block height `3,526,800`;
-1. Update the binary to `v3.3.2` and start the node again.
+| Block height          | Binary Version        | Instruction |
+| --------------------- | --------------------- | -------------------------------------------- |
+| `1 - 922,363`         | `chain-main_1.2.1`    | Start the node with the older binary version |
+| `922,363`             | `chain-main_2.0.1`    | When it reaches the target block height `922,363` (Canis Major), update the binary and restart |
+| `922,363 - 3,526,800` | `chain-main_3.3.3`    | When it reaches the target block height `3,526,800` (Draco), update the binary and restart |
+| `>3,526,800`          | `chain-main_3.3.5`    | Update the binary to the latest `3.3.5` (incl. DragonBerry, IAVL Migration) binary and restart* |
+| `>3,526,800`          | `chain-main_3.3.9`    | Update the binary to the latest `3.3.9` (incl. IAVL upgrade) binary and restart* |
```

Comment:
Could suggest `3.3.9` at the block `3,526,800` but would have to add the IAVL config for that.